api/graphql/schema.graphql

```diff
@@ -627,6 +631,69 @@ type TalentNominationEvent {
   developmentPrograms: [DevelopmentProgram!] @belongsToMany
 }
 
+type TalentNomination {
+  id: UUID!
+  submittedSteps: [TalentNominationStep!] @rename(attribute: "submitted_steps")
+  talentNominationEvent: TalentNominationEvent! @belongsTo
+  submittedAt: DateTime @rename(attribute: "submitted_at")
+  submitter: BasicGovEmployeeProfile!
+    @belongsTo
+    @canQuery(ability: "viewAnyBasicGovEmployeeProfile", model: "User")
+  submitterRelationshipToNominator: LocalizedTalentNominationSubmitterRelationshipToNominator
+    @rename(attribute: "submitter_relationship_to_nominator")
+  submitterRelationshipToNominatorOther: String
+    @rename(attribute: "submitter_relationship_to_nominator_other")
+  nominator: BasicGovEmployeeProfile
+    @belongsTo
+    @canQuery(ability: "viewAnyBasicGovEmployeeProfile", model: "User")
```

Comment:
I would have preferred to use canResolved to also verify that the returned model is a government employee but that seems to always fail when null.  So I had to add a scope to the relationship instead, which seems less clean to me.